workitem/field_definition.go

```diff
@@ -7,28 +7,29 @@ import (
 
 	"strings"
 
+	"github.com/davecgh/go-spew/spew"
 	"github.com/fabric8-services/fabric8-wit/convert"
 
 	errs "github.com/pkg/errors"
 )
 
 // constants for describing possible field types
 const (
-	KindString            Kind = "string"
-	KindInteger           Kind = "integer"
-	KindFloat             Kind = "float"
-	KindInstant           Kind = "instant"
-	KindDuration          Kind = "duration"
-	KindURL               Kind = "url"
-	KindIteration         Kind = "iteration"
-	KindWorkitemReference Kind = "workitem"
```

Comment:
what happend to `KindWorkitemReference` ?

Reply:
Oh, yes. That is gone on purpose but I forgot to mention it in the Description. We never used it and it had a wrong type (int) as well IIRC.

Reply:
ok, good